Fran 4 26

Frantastic74 opened the pull request.

opbld34 commented:
Docs Build status updates of commit _[2ef3a97](https://github.com/MicrosoftDocs/mixed-reality/pull/1070/commits/2ef3a97384a0867cb57912a88452de7072a54e8a)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-a65e502d)

<a id="a65e502d"></a>
###
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/MicrosoftDocs/mixed-reality, working directory: W:\03nx-s, target branch: master, target branch commit: 19b697854471a757548f201b7e13560f413eb970, source commit: 2ef3a97384a0867cb57912a88452de7072a54e8a.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C2%5C45d83296-8fc9-fec5-a3a5-a2e97fb9db25%5CPullRequest%5C201905021624298394-1070%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9vtZrIZiDJapLV7JteEl4f%2BPH5zv1DQB%2FT4fTxrUoGI%3D&st=2019-05-02T16%3A19%3A49Z&se=2019-06-02T16%3A24%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Frantastic74 commented:
I fixed the html tables to markdown and fixed all Md issues - There shouldn't be conflicting files unless those links have changes and I'll check. I edited this doc earlier but pushed to Casey's branch. Please review.

jessemcculloch commented:
Old PR, will approve latest version
